backend/fms_core/template_importer/importers/sample_qc.py

```diff
@@ -0,0 +1,85 @@
+from fms_core.models import Process, Protocol, PropertyType
+
+from ._generic import GenericImporter
+from fms_core.template_importer.row_handlers.sample_qc import SampleQCRowHandler
+
+from .._utils import float_to_decimal_and_none
+
+class SampleQCImporter(GenericImporter):
+    SHEETS_INFO = [
+        {
+            'name': 'SampleQC',
+            'headers': ['Sample Container Barcode', 'Sample Container Coord', 'Initial Volume (uL)',
+                        'Measured Volume (uL)', 'Volume Used (uL)', 'Concentration (ng/uL)', 'NA Quantity (ng)',
+                        'RIN (for RNA only)', 'Electrophoresis Instrument', 'Quality Flag', 'Quantitation Instrument',
+                        'Quantity Flag', 'QC Date', 'Comment']
+        },
+    ]
+
+    def __init__(self):
+        super().__init__()
+        self.initialize_data_for_template()
+
+    def initialize_data_for_template(self):
+        #Get protocol for SampleQC
+        protocol = Protocol.objects.get(name='Sample Quality Control')
+
+        #Preload data
+        self.preloaded_data = {'process': None, 'process_properties': {}}
+
+        self.preloaded_data['process'] = Process.objects.create(protocol=protocol,
+                                                                comment='Sample Quality Control (imported from template)')
+
+        # Preload PropertyType objects for the sample qc in a dictionary for faster access
+        try:
+            self.preloaded_data['process_properties'] = { property.name: {'property_type_obj': property } for property in
```

Comment:
i would change process_properties to process_measurement_properties to make it clear those are not tied to a process.